### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-6-14

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
- functionality is working 
- well written code 
- good to merge

File: social-app/src/Component/ProfileModal/ProfileModal.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.profileModal_container{
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+     height: 100%;
+    background-color: #00000099;
```
Comment (MarkVed17) [inline social-app/src/Component/ProfileModal/ProfileModal.css:7]: It would be good to use CSS variables.

**Review by MarkVed17 — APPROVED** [review]
Rest all looks good to proceed!

File: social-app/src/Component/ProfileModal/ProfileModal.css (hunk 1 above)
Comment (aniketsingh13) [reply social-app/src/Component/ProfileModal/ProfileModal.css:7]: will do that

**Pull request merged**